logo.phtml, @var $storeName not defined

Amadeco opened the pull request:
Line 20 :

```
<a href="<?php echo $block->getUrl(''); ?>" class="logo navbar-brand" title="<?php /* @escapeNotVerified */ echo $storeName ?>">
```

replace

```
<a href="<?php echo $block->getUrl(''); ?>" class="logo navbar-brand" title="<?php /* @escapeNotVerified */ echo $block->getLogoAlt() ?>">
```

bartdelvaux committed:
Merge remote-tracking branch 'upstream/develop' into develop

Commit:
MAGETWO-43789: "Select All" button does not work in Cache Management grid page

zanilee committed:
MAGETWO-44225: Items requiring attention section of minicart is not updated after items removal

Commit:
MAGETWO-44757: JS error on checkout for guest with virtual product

Commit:
MAGETWO-44757: JS error on checkout for guest with virtual product

Commit:
MAGETWO-44753: "Require Customer To Be Logged In To Checkout" Config Setting is Useless

Commit:
MAGETWO-42380: "Create Link Widget" Admin page UI issues

Commit:
MAGETWO-42380: "Create Link Widget" Admin page UI issues

Commit:
MAGETWO-42380: "Create Link Widget" Admin page UI issues

Commit:
MAGETWO-42380: "Create Link Widget" Admin page UI issues

Commit:
MAGETWO-42380: "Create Link Widget" Admin page UI issues

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

zanilee committed:
MAGETWO-45096: Doubled delimiters are present on "view Grouped Product" Storefront page in responsive

- Files formatting

zanilee committed:
MAGETWO-45096: Doubled delimiters are present on "view Grouped Product" Storefront page in responsive

zanilee committed:
MAGETWO-45129: Copyright isn't visible on "Login" Admin page

zanilee committed:
MAGETWO-45094: Split button content partly overlaps filter content in Admin

Commit:
MAGETWO-45177: Improve indents in the less files according less code standards

Commit:
MAGETWO-45177: Improve indents in the less files according less code standards

Commit:
Merge remote-tracking branch 'origin/develop' into MAGETWO-45177

zanilee committed:
MAGETWO-45101: Products with wide base images overlap each other on "view Wish List" Storefront page

Commit:
MAGETWO-45177: Improve indents in the less files according less code standards

Commit:
MAGETWO-45097: Long SKU breaks "view Product" Storefront page layout if responsive

Commit:
MAGETWO-45009: Specific Countries option take no effect on Zero Subtotal Checkout

Commit:
MAGETWO-45145: The Form on "Login" Admin page is displayed invalidly aligned in IE

Commit:
MAGETWO-45009: Specific Countries option take no effect on Zero Subtotal Checkout

- added changes to unit test

Commit:
MAGETWO-45254: Checkout Place Order action returns promise with irreplaceable redirect to checkout/onepage/success/

Commit:
MAGETWO-45254: Checkout Place Order action returns promise with irreplaceable redirect to checkout/onepage/success/

Commit:
MAGETWO-45297: [Sample Data] Update README.md file

Commit:
MAGETWO-44532: Price section on "view Product" Storefront page isn't fully visible on tablet devices

Commit:
MAGETWO-45297: [Sample Data] Update README.md file

Commit:
MAGETWO-45305: Form styling is broken on New Email Template page

Commit:
MAGETWO-44874: Message about your required attention product isn't displayed

Commit:
MAGETWO-45053: Horizontal scroll is present on CMS pages on mobile

Commit:
MAGETWO-45134: Admin Navigation Menu contains items delimiter displayed if correspondent item is selected

Commit:
MAGETWO-45090: PHP Settings Check error is displayed outside its column on Web Setup

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45297' into develop

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

Commit:
MAGETWO-45097: Long SKU breaks "view Product" Storefront page layout if responsive

Commit:
Merge remote-tracking branch 'mainline/develop' into develop

Commit:
MAGETWO-45489: Create integration tests for Paypal Hosted Pro solution

 - Added integration tests for Hostedpro::initialize() and Request::setOrder()

Commit:
MAGETWO-45489: Create integration tests for Paypal Hosted Pro solution

 - Fixed failed static tests

Commit:
MAGETWO-45098: Long Category name breaks Navigation Menu Storefront layout if responsive

Commit:
MAGETWO-45090: PHP Settings Check error is displayed outside its column on Web Setup

Commit:
MAGETWO-45442: Navigation Menu is cut in IE 11

Commit:
MAGETWO-45500: Date range fields are cut on Sales Report page

Commit:
MAGETWO-45546: Multi select in Cache & Index grids not clickable as expected

Commit:
MAGETWO-45442: Navigation Menu is cut in IE 11

irenelagno committed:
MAGETWO-44192: Tax rules in catalog does not work according to persisted customer data

Commit:
Merge remote-tracking branch 'mainline/develop' into develop

irenelagno committed:
MAGETWO-44192: Tax rules in catalog does not work according to persisted customer data

irenelagno committed:
MAGETWO-44192: Tax rules in catalog does not work according to persisted customer data

bartdelvaux committed:
Merge remote-tracking branch 'upstream/master' into develop

bartdelvaux committed:
Make the strings of checkout steps translatable
The strings "Shipping" and "Review and Payments" were not translatable on the checkout page.

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

Commit:
MAGETWO-45489: Create integration tests for Paypal Hosted Pro solution

 - Refactored Hostedpro request
 - Moved price formatting to separated helper trait

Commit:
MAGETWO-44998: Save in address book checkbox is unchecked by default on billing address form

fixed new address form default value

Commit:
MAGETWO-45489: Create integration tests for Paypal Hosted Pro solution

 - Fixed failed static tests
 - Refactored code to minimize NPath complexity

Commit:
MAGETWO-45489: Create integration tests for Paypal Hosted Pro solution

 - Added parameter name to formatPrice() method

Commit:
MAGETWO-39065: [PO] There is no information about applied gift card account amount on payment step

zanilee committed:
MAGETWO-45598: Order status and Cart Price Rule fields look not aligned on Report pages

irenelagno committed:
MAGETWO-45059: Behaviour of JS validation for guest email form field

Commit:
Merge remote-tracking branch 'mainline/develop' into develop

# Conflicts:
#	app/code/Magento/SampleData/README.md

Commit:
MAGETWO-45652: Can't Deploy Sample Data After "composer create-project"

zanilee committed:
MAGETWO-45095: Product base images aren't centered in Wish List on Storefront

- Formatting and sorting

Commit:
MAGETWO-42814: [UI] Closed and Upcoming Events are displayed unbelievably giant on mobile devices

Commit:
MAGETWO-45088: Empty space is displayed on "Advanced Search Results" Storefront page

Commit:
MAGETWO-45088: Empty space is displayed on "Advanced Search Results" Storefront page

irenelagno committed:
MAGETWO-44232: Estimator does not work in IE with enabled Minify JavaScript Files

Commit:
MAGETWO-45657: There is no email validation if checkout by Gift Registry link

Commit:
Merge remote-tracking branch 'origin' into MAGETWO-45145

Commit:
Merge remote-tracking branch 'mainline/develop' into bugs

Commit:
MAGETWO-45088: Empty space is displayed on "Advanced Search Results" Storefront page

Commit:
MAGETWO-45088: Empty space is displayed on "Advanced Search Results" Storefront page

Commit:
MAGETWO-44998: Save in address book checkbox is unchecked by default on billing address form

Commit:
MAGETWO-42814: [UI] Closed and Upcoming Events are displayed unbelievably giant on mobile devices

zanilee committed:
MAGETWO-45095: Product base images aren't centered in Wish List on Storefront

irenelagno committed:
MAGETWO-45665: There is no ability to sign out if Clear Persistence on Log Out = No

Commit:
MAGETWO-45088: Empty space is displayed on "Advanced Search Results" Storefront page

Commit:
MAGETWO-43340: JS loader is displayed outside the main content area in Admin

Commit:
MAGETWO-42814: [UI] Closed and Upcoming Events are displayed unbelievably giant on mobile devices

irenelagno committed:
MAGETWO-44192: Tax rules in catalog does not work according to persisted customer data

Commit:
Merge pull request #194 from magento-nord/develop

[NORD] SampleData updates

Commit:
MAGETWO-45672: Broken checkout flow

added unit test

Commit:
MAGETWO-45757: CSRF Vulnerability on Cart Checkout

Commit:
Merge remote-tracking branch 'mainline/develop' into bugs

Commit:
MAGETWO-43159: [Dev experience] Incorrect binding for cart().extra_actions

Commit:
MAGETWO-42103: Minicart does not clear after completing an order via Paypal

Commit:
MAGETWO-45100: Create registry Storefront page heading lost its bottom padding if responsive

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45546' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45145' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45094' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-44532' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45101' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45100' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45097' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45098' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45305' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45096' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45500' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45442' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45090' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45053' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-42380' into PR-1

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45134' into PR-1

Commit:
MAGETWO-45129: Copyright isn't visible on "Login" Admin page

- Merge remote-tracking branch 'origin/MAGETWO-45129' into PR-1

- Conflicts:
	- app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/pages/_login.less

Commit:
MAGETWO-45177: Improve indents in the less files according less code standards

- Merge remote-tracking branch 'origin/MAGETWO-45177' into PR-1

- Conflicts:
	- app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/pages/_login.less
	- app/design/adminhtml/Magento/backend/web/app/setup/styles/less/pages/_readiness-check.less
	- app/design/adminhtml/Magento/backend/web/css/source/_structure.less
	- app/design/frontend/Magento/blank/Magento_GroupedProduct/web/css/source/_module.less
	- app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less
	- app/design/frontend/Magento/luma/Magento_GiftRegistry/web/css/source/_module.less
	- app/design/frontend/Magento/luma/Magento_GroupedProduct/web/css/source/_module.less

Commit:
MAGETWO-45672: Broken checkout flow

fixed annotations

Commit:
MAGETWO-44131: Sometimes "Estimate Shipping & Tax" section overlaps Newsletter Subscription form in Shopping Cart

Commit:
Merge pull request #197 from magento-folks/bugs

[Folks] Bugfix

Commit:
MAGETWO-45177: Improve indents in the less files according less code standards

- Merge remote-tracking branch 'origin/develop' into PR-1

- Conflicts:
	- app/design/frontend/Magento/blank/Magento_AdvancedCheckout/web/css/source/_module.less
	- app/design/frontend/Magento/luma/Magento_AdvancedCheckout/web/css/source/_module.less

Commit:
Merge branch 'pr-checkout-step-translation' of https://github.com/bartdelvaux/magento2 into MAGETWO-45916

spphoenix committed:
PRD-26: Catalog Search Index 1 - basic indexation mechanism

Commit:
Merge pull request #199 from magento-webdev/PR-1

[WebDev] Bugfixes

Commit:
MAGETWO-45842: Remove icon in product compare not displayed correctly

Commit:
MAGETWO-45703: [JS] Search by criteria for UI components registry

- Implement components lookup
- Remove redundant configuration

Commit:
MAGETWO-45701: [JS] RequireJS resolver (for Loader widget)

- Implement tracking of requirejs's loading state
- Apply it on form and grid components

Commit:
MAGETWO-45702: [JS] Caching server requests for UI Listing component

- Implement requests caching

Commit:
MAGETWO-45703: [JS] Search by criteria for UI components registry

- Remove unused files

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Implement knockout's alternative template syntax
- Apply it in Ui module templates

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Fix code style issues

Commit:
MAGETWO-45701: [JS] RequireJS resolver (for Loader widget)

- Remove resolver usage from button widget.

Commit:
Merge branch 'develop' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

Commit:
MAGETWO-45703: [JS] Search by criteria for UI components registry

- Extend xsd

Commit:
MAGETWO-45703: [JS] Search by criteria for UI components registry

- Add missing paging component

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Make toolbar component non-sticky by default

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

 - remove swatches js files from blacklist

Commit:
MAGETWO-45703: [JS] Search by criteria for UI components registry

- Rewrite xsd rule

Commit:
MAGETWO-46013: Issue with a legacy test

- Fixed failing static test: Magento\Test\Legacy\WordsTest::testWords. Excluded composer.lock from the checked file list

Commit:
Merge remote-tracking branch 'origin/MAGETWO-46013-L3-build-issue' into PR_Branch

Commit:
Merge pull request #203 from magento-ogre/PR_Branch

[Ogres] Bug fixes

Commit:
Merge branch 'develop' of github.corp.magento.com:magento2/magento2ce into MAGETWO-45916

Vinai commented:
Thank you for your contribution!  
In order to further process your PR we will have to wait for the travis build infrastructure to be fixed for the tests to pass. This is currently being worked on.  
In the meanwhile, the contributor license agreement needs to be signed by you, in order for us to be able to process the PR. Please follow the link "Details" below to do so.

spphoenix committed:
PRD-26: Catalog Search Index 1 - basic indexation mechanism

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Add element identifier for functional tests

Commit:
MDVA-24: Problems when importing product data with store_view_code

 - fixed bugs with import

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Add missing attributes
- Remove unnecessary template

Commit:
MAGETWO-45701: [JS] RequireJS resolver (for Loader widget)

- Code style fix

Commit:
Merge branch 'develop' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Fix scope issue

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Rename custom scope attribute

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

Commit:
MAGETWO-42901: [UI] Features Platinum Partners Page - assistance

- Squashed commit

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

Commit:
Merge commit 'refs/pull/45/head' of https://github.com/magento/magento2ce into PR_Branch_ES

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Temporary disable requests caching

Commit:
MDVA-24: Problems when importing product data with store_view_code

 - fixed bugs with import

Commit:
Merge branch 'develop' into MDVA-24

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Change components xsd

Commit:
MAGETWO-42418: [TD] Color Swatches javascript code review fixes

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Add active area flag for functional tests
- Enable requests caching

Commit:
MAGETWO-45583: Disabling Shipping methods modules (UPS, USPS, DHL, Fedex)

Commit:
Merge pull request #202 from magento-folks/MAGETWO-45916

[Folks] Public GitHub Pull Request

dkvashninbay committed:
MAGETWO-46040: Cannot change payment method in case of failure

Commit:
MAGETWO-46011: The empty mandatory fields are not displayed for a user, after clicking on Save button

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45842' into PR-4

Commit:
Merge remote-tracking branch 'origin/MAGETWO-43789' into PR-4

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45598' into PR-4

Commit:
Merge remote-tracking branch 'origin/MAGETWO-43340' into PR-4

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45088' into PR-4

Commit:
MAGETWO-42814: [UI] Closed and Upcoming Events are displayed unbelievably giant on mobile devices

- Merge remote-tracking branch 'origin/MAGETWO-42814' into PR-4

- Conflicts:
	- app/design/frontend/Magento/blank/Magento_CatalogEvent/web/css/source/_module.less

Commit:
Merge remote-tracking branch 'origin/MAGETWO-42901-SQ' into PR-4

Commit:
MAGETWO-45701: [JS] RequireJS resolver (for Loader widget)

- Apply resolver on checkout page

Commit:
MAGETWO-44131: Sometimes "Estimate Shipping & Tax" section overlaps Newsletter Subscription form in Shopping Cart

Commit:
Merge branch 'develop' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

Commit:
Merge remote-tracking branch 'origin/develop' into swatches-cr

anton-huz committed:
MAGETWO-46107: Checkboxes don't send their value on Customer Edit Form

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Add missing node type

Commit:
MAGETWO-45702: [JS] Caching server requests for UI Listing component

- Enable caching

Commit:
Merge remote-tracking branch 'mainline/develop' into develop

Commit:
Merge pull request #209 from magento-troll/MAGETWO-42418

MAGETWO-42418: [TD] Color Swatches javascript code review fixes

Commit:
Merge remote-tracking branch 'origin/develop' into PR-4

Commit:
MAGETWO-45180: [Refactoring] Accept Credit Card Payments using Braintree for StoreFront and Backend

 - Added new implementation of Braintree payment solution

anton-huz committed:
MAGETWO-46107: Checkboxes don't send their value on Customer Edit Form

Commit:
MAGETWO-45702: [JS] Caching server requests for UI Listing component

- Add cached request delay

Commit:
MAGETWO-45701: [JS] RequireJS resolver (for Loader widget)

Commit:
Merge branch 'MAGETWO-44467' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

Commit:
Merge branch 'develop' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

Commit:
Merge pull request #211 from magento-webdev/PR-4

[WebDev] Bugfixes

Commit:
Merge remote-tracking branch 'mainline/develop' into PR_Branch_ES

Commit:
MAGETWO-45180: [Refactoring] Accept Credit Card Payments using Braintree for StoreFront and Backend

 - Changed event triggering to direct function call

Commit:
Merge branch 'develop' into MDVA-24

Commit:
Merge remote-tracking branch 'mainline/develop' into develop

Commit:
Merge branch 'develop' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

Conflicts:
	app/code/Magento/Marketplace/view/adminhtml/templates/index.phtml

Commit:
MAGETWO-45702: [JS] Caching server requests for UI Listing component

- Increase delay time
- Disable caching on configurable products listing

Commit:
MAGETWO-45702: [JS] Caching server requests for UI Listing component

- Disable caching on product attributes

slavvka committed:
Merge pull request #213 from magento-sparta/MDVA-24

[SUPPORT] MDVA-24: Problems when importing product data with store_view_code

Commit:
Merge remote-tracking branch 'mainline/develop' into develop

Commit:
MAGETWO-46063: [Dev Experience] Checkout Agreements resource model data inconsistency

Commit:
MAGETWO-45702: [JS] Caching server requests for UI Listing component

- Decrease update delay
- Fix wrong request parameters issue

Commit:
MAGETWO-44467: Magento HTML templates basic format

- Comments

Commit:
Merge branch 'develop' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

Commit:
MAGETWO-46091: Estimate-shipping-methods Service is Unavailable for Admin

Dell- committed:
MAGETWO-37552: [UI] Handling Invalid Less During PHP Compilation (part 2)

- Removed catching exceptions

Commit:
MAGETWO-45180: [Refactoring] Accept Credit Card Payments using Braintree for StoreFront and Backend

- Change convertation method object to array via type casting in HTTP/Client
- Remove unnessessary constant and variable in response handler

Commit:
Merge pull request #216 from magento-nord/develop

[NORD] Bugs

Commit:
Merge remote-tracking branch 'mainline/develop' into PR_Branch_ES

Commit:
Merge pull request #206 from magento-ogre/PR_Branch_ES

[Phoenix Media - Ogres] Accept Phoenix Media Sprint 2 code for Elastic Search Adapter

Commit:
MAGETWO-44852: [Folks] TAF activity S77

added constraint for minicart

Commit:
MAGETWO-45180: [Refactoring] Accept Credit Card Payments using Braintree for StoreFront and Backend

 - Refactored code related to code review
 - Added event triggering for single configured payment on Backend
 - Added check for empty Braintree response

Commit:
Merge branch 'MAGETWO-45180' of github.corp.magento.com:magento-mpi/magento2ce into MAGETWO-45180

Dell- committed:
Merge branch 'develop' of github.corp.magento.com:magento2/magento2ce into MAGETWO-37552

Commit:
Merge remote-tracking branch 'mainline/develop' into bugs

Commit:
Merge branch 'develop' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

dkvashninbay committed:
Merge pull request #218 from magento-mpi/MAGETWO-37552

[MPI] Bugs

Commit:
MAGETWO-45701: [JS] RequireJS resolver (for Loader widget)

- Apply resolver on checkout steps

Commit:
Merge branch 'develop' of https://github.corp.magento.com/magento-vanilla/magento2ce into MAGETWO-44467

Commit:
MAGETWO-45180: [Refactoring] Accept Credit Card Payments using Braintree for StoreFront and Backend

 - Refactored code related to code review
 - Added @todo annotations for refactoring after changes in payment and sales module

Commit:
Merge branch 'develop' of github.corp.magento.com:magento2/magento2ce into MAGETWO-45180

Commit:
MAGETWO-45180: [Refactoring] Accept Credit Card Payments using Braintree for StoreFront and Backend

 - Fixed failed static tests

Commit:
MAGETWO-44852: [Folks] TAF activity S77

updates
CR changes

zanilee committed:
MAGETWO-45095: Product base images aren't centered in Wish List on Storefront

- Merge remote-tracking branch 'origin/develop' into MAGETWO-45095
- Conflicts:
- app/design/frontend/Magento/blank/Magento_Catalog/web/css/source/module/_listings.less
- app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less
- app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less

anton-huz committed:
Merge pull request #219 from magento-vanilla/MAGETWO-44467

[Vanilla] Tasks

Commit:
MAGETWO-46000: Terms and Conditions Checkbox Text is Hard-Coded as a Title in the Terms and Conditions Template

Commit:
MAGETWO-46206: Fix random fails of Magento\Checkout\Test\TestCase\OnePageCheckoutTest functional test

 fixed javascript error

Dell- committed:
MAGETWO-46207: Integration Vault of payment solution

- Integration

zanilee committed:
MAGETWO-45095: Product base images aren't centered in Wish List on Storefront

Dell- committed:
MAGETWO-46207: Integration Vault of payment solution

- CR

Commit:
MAGETWO-46166: Unit Tests Introduce Dependency on Non-existing Module

Commit:
MAGETWO-46207: Integration Vault of payment solution

- Fixed tests

Commit:
Merge branch 'MAGETWO-42231-PR' of github.corp.magento.com:magento-mpi/magento2ce into MAGETWO-42231-PR

zanilee committed:
MAGETWO-46052: Wish Lists UI issues

Commit:
Merge remote-tracking branch 'mainline/develop' into bugs

Commit:
Merge remote-tracking branch 'origin/MAGETWO-44131' into PR

zanilee committed:
MAGETWO-45910: Bottom padding is missed on Search results Storefront page

Commit:
MAGETWO-45180: [Refactoring] Accept Credit Card Payments using Braintree for StoreFront and Backend

 - Updated card type validation for Braintree
 - Updated tips for config section

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45910-new' into PR

Commit:
Merge remote-tracking branch 'origin/MAGETWO-45095' into PR

zanilee committed:
MAGETWO-46052: Wish Lists UI issues

- Merge remote-tracking branch 'origin/MAGETWO-46052-new' into PR
- Conflicts:
- app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less

Commit:
MAGETWO-46206: Fix random fails of Magento\Checkout\Test\TestCase\OnePageCheckoutTest functional test

 - fixed fields order for shipping address form

Commit:
MAGETWO-46416: Shipping estimation on Shopping Cart doesn't work randomly

added dependency to estimation block

Commit:
MAGETWO-46206: Fix random fails of Magento\Checkout\Test\TestCase\OnePageCheckoutTest functional test

 - CR changes

Commit:
Merge pull request #226 from magento-webdev/PR

[WebDev] Bugfixes

Commit:
Merge branch 'develop' of github.corp.magento.com:magento2/magento2ce into bugs

Commit:
MAGETWO-46268: Contribute S73 to mainline

- Fix test and update composer .lock and .json files

Commit:
Merge branch 'develop' of github.corp.magento.com:magento2/magento2ce into MAGETWO-45180-PR

Commit:
Merge branch 'develop' of github.corp.magento.com:magento2/magento2ce into MAGETWO-42231-PR

Commit:
Merge branch 'MAGETWO-45180-PR' of github.corp.magento.com:magento-mpi/magento2ce into MPI-S73-PR

Commit:
MAGETWO-46268: Contribute S73 to mainline

- Fixed static tests

Commit:
Merge branch 'MAGETWO-42231-PR' of github.corp.magento.com:magento-mpi/magento2ce into MPI-S73-PR

Conflicts:
	composer.lock

Commit:
MAGETWO-46268: Contribute S73 to mainline

- Updated composer.lock file

Commit:
MAGETWO-46268: Contribute S73 to mainline

- Fixed static test

Commit:
Merge pull request #224 from magento-folks/bugs

[Folks] Bugfix

dkvashninbay committed:
Merge remote-tracking branch 'main/develop' into MPI-S73-PR

dkvashninbay committed:
Merge pull request #229 from magento-mpi/MPI-S73-PR

[MPI] Bug fixes and tasks
